target/linux/ath79/image/generic.mk

```diff
@@ -205,47 +219,53 @@ TARGET_DEVICES += comfast_cf-e5
 
 define Device/devolo_dvl1200e
   ATH_SOC := qca9558
-  DEVICE_TITLE := devolo WiFi pro 1200e
+  DEVICE_VENDOR := Devolo
```

Comment:
devolo seems to always use lower-case for its company name. I would stick to that (also for the following).